--- a/app/Sections/Hero.tsx
+++ b/app/Sections/Hero.tsx
@@ -16,7 +16,7 @@ export default function Hero() {
   return (
     <div
       id="hero"
-      className="w-full h-[86vh] bg-black/20 flex flex-col justify-center items-center relative overflow-hidden px-4 pb-0 sm:px-6 lg:px-0"
+      className="w-full min-h-[86vh] h-auto bg-black/20 flex flex-col justify-center items-center relative overflow-hidden px-4 pb-0 sm:px-6 lg:px-0"
     >
       <section className="beams w-full h-full absolute pointer-events-none">
         <Beams
@@ -30,7 +30,7 @@ export default function Hero() {
           rotation={30}
         />
       </section>
-      <div className="text-center relative pointer-events-auto w-[80%] mx-auto">
+      <div className="text-center relative pointer-events-auto w-full sm:w-[90%] md:w-[80%] mx-auto">
         {/* blue text */}
         <aside className="flex flex-col items-center justify-center gap-6">

